compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -439,7 +439,7 @@ final class Conversions(
     val tx = incomplete.transaction
 
     val builder = proto.PartialTransaction.newBuilder
-      .addAllNodes(tx.nodes.map(convertNode).asJava)
+      .addAllNodes(tx.nodes.map(convertIncompleteNode(incomplete.locationInfo)).asJava)
```

Comment:
I think this is slightly misleading. The nodes are not necessarily incomplete only some might be. Maybe `convertPtxNode`?

Reply:
`convertIncompleteTransactionNode`